.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+**'
```

Comment:
Do we need two stars? is this actually a regex pattern?

It's close I guess:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Maybe, we just need `*` at the end...

Reply:
Yeah - one star is probably better